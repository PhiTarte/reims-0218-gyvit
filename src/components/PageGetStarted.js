import React from "react"
import { Row, Col, Container, Form } from "reactstrap"
import PageGetStartedForm from "./PageGetStartedForm"
import GetStartedSignInContent from "./GetStartedSignInContent"

const PageGetStarted = () => (
  <Container fluid>
    <Form>
      <Row>
        <Col md="4">
          <PageGetStartedForm />
        </Col>
        <Col md="8">
          <GetStartedSignInContent />
        </Col>
      </Row>
    </Form>
  </Container>
)

export default PageGetStarted
