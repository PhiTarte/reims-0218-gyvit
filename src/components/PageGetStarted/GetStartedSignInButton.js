import React from "react"
import { Button } from "reactstrap"

const GetStartedSignInButton = () => (
  <div>
    <Button
      style={{
        width: "192px",
        height: "54px",
        "border-radius": "100px",
        "background-image": "linear-gradient(to left, #57aad1, #7a57d1)"
      }}
      color="info"
    >
      Continue
    </Button>
  </div>
)

export default GetStartedSignInButton
