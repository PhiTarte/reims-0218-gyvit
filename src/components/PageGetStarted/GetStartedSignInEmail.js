import React from "react"
import { FormGroup, Label, Input } from "reactstrap"

export default class GetStartedSignInEmail extends React.Component {
  render() {
    return (
      <div
        style={{
          "margin-bottom": "40px",
          "margin-top": "60px"
        }}
      >
        <FormGroup>
          <Label
            style={{
              "margin-top": "30px",
              width: "33px",
              height: "12px",
              fontSize: "12px",
              fontWeight: "normal",
              fontStyle: "normal",
              "letter-spacing": "normal",
              "text-align": "left",
              color: "#9da5c9"
            }}
            for="email"
          >
            Email{" "}
          </Label>
          <Input type="email" name="email" id="IdEmail" placeholder="Email" />
        </FormGroup>
        <FormGroup>
          <Label
            style={{
              "margin-top": "30px",
              width: "33px",
              height: "12px",
              fontSize: "12px",
              fontWeight: "normal",
              fontStyle: "normal",
              "letter-spacing": "normal",
              "text-align": "left",
              color: "#9da5c9"
            }}
            for="email"
          >
            Name{" "}
          </Label>
          <Input type="email" name="name" id="IdEmail" placeholder="Name" />
        </FormGroup>
        <FormGroup>
          <Label
            style={{
              "margin-top": "30px",
              width: "75px",
              height: "12px",
              fontSize: "12px",
              fontWeight: "normal",
              fontStyle: "normal",
              "letter-spacing": "normal",
              "text-align": "left",
              color: "#9da5c9"
            }}
            for="password"
          >
            Password
          </Label>
          <Input
            type="password"
            name="password"
            id="IdPassword"
            placeholder="Password"
          />
        </FormGroup>
      </div>
    )
  }
}
