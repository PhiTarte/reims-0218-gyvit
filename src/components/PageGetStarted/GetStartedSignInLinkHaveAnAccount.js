//TODO CREATE SIGN IN TITLE
import React from "react"
import { NavLink } from "reactstrap"

const GetStartedSignInLinkHaveAnAccount = () => (
  <div
    style={{
      marginTop: "120px",
      width: "300px",
      height: "16px",
      fontSize: "16px",
      fontWeight: "normal",
      fontStyle: "normal",
      letterSpacing: "normal",
      textAlign: "left",
      color: "#3b425d"
    }}
  >
    <p>
      Déjà un compte ?
      <NavLink
        href="#"
        style={{
          display: "inline"
        }}
      >
        Se connecter
      </NavLink>
    </p>
  </div>
)

export default GetStartedSignInLinkHaveAnAccount
