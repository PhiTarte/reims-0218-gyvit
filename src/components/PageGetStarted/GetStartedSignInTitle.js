//TODO CREATE SIGN IN TITLE
import React from "react"

const GetStartedSignInTitle = () => (
  <div>
    <p
      style={{
        height: "24px",
        "font-size": "24px",
        "font-weight": "normal",
        "font-style": "normal",
        "font-stretch": "normal",
        "line-height": "normal",
        "letter-spacing": "normal",
        "text-align": "left",
        color: "#3b425d"
      }}
    >
      Get started absolutely free
    </p>
    <p
      style={{
        width: "300px",
        height: "18px",
        fontSize: "18px",
        fontWeight: "normal",
        fontStyle: "normal",
        "letter-spacing": "normal",
        "text-align": "left",
        color: "#9da5c9"
      }}
    >
      Free forever. No credit card needed.
    </p>
  </div>
)

export default GetStartedSignInTitle
