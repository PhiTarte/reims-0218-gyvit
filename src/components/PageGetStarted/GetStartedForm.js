import React from "react"
import { Container } from "reactstrap"
import GetStartedSignInEmail from "./GetStartedSignInEmail"
import GetStartedSignInTitle from "./GetStartedSignInTitle"
import GetStartedSignInButton from "./GetStartedSignInButton"
import GetStartedSignInLinkHaveAnAccount from "./GetStartedSignInLinkHaveAnAccount"

const GetStartedForm = () => (
  <Container
    style={{
      paddingTop: "25%",
      marginLeft: "30px",
      backgroundColor: "#ffffff",
      marginRight: "50px"
    }}
  >
    <GetStartedSignInTitle />
    <GetStartedSignInEmail />
    <GetStartedSignInButton />
    <GetStartedSignInLinkHaveAnAccount />
  </Container>
)

export default GetStartedForm
