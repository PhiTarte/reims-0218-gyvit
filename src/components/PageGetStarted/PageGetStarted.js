import React from "react"
import { Row, Col, Container, Form } from "reactstrap"
import GetStartedForm from "./GetStartedForm"
import GetStartedSignInContent from "./GetStartedSignInContent"

const PageGetStarted = () => (
  <Container fluid>
    <Form>
      <Row>
        <Col md="4">
          <GetStartedForm />
        </Col>
        <Col md="8">
          <GetStartedSignInContent />
        </Col>
      </Row>
    </Form>
  </Container>
)

export default PageGetStarted
