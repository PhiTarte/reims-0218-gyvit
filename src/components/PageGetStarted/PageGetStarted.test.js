import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import PageGetStarted from "./PageGetStarted"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<PageGetStarted />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<PageGetStarted />).toJSON()
  expect(tree).toMatchSnapshot()
})
