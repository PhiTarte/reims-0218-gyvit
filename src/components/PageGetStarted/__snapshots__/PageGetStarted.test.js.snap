// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<div
  className="container-fluid"
>
  <form
    className=""
  >
    <div
      className="row"
    >
      <div
        className="col-md-4"
      >
        <div
          className="container"
          style={
            Object {
              "background-color": "#ffffff",
              "margin-left": "30px",
              "margin-right": "50px",
              "padding-top": "25%",
            }
          }
        >
          <form
            className=""
          >
            <div>
              <p
                style={
                  Object {
                    "color": "#3b425d",
                    "font-size": "24px",
                    "font-stretch": "normal",
                    "font-style": "normal",
                    "font-weight": "normal",
                    "height": "24px",
                    "letter-spacing": "normal",
                    "line-height": "normal",
                    "text-align": "left",
                  }
                }
              >
                Get started absolutely free
              </p>
              <p
                style={
                  Object {
                    "color": "#9da5c9",
                    "fontSize": "18px",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                    "height": "18px",
                    "letter-spacing": "normal",
                    "text-align": "left",
                    "width": "300px",
                  }
                }
              >
                Free forever. No credit card needed.
              </p>
            </div>
            <div
              style={
                Object {
                  "margin-bottom": "40px",
                  "margin-top": "60px",
                }
              }
            >
              <div
                className="form-group"
              >
                <label
                  className=""
                  htmlFor="email"
                  style={
                    Object {
                      "color": "#9da5c9",
                      "fontSize": "12px",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                      "height": "12px",
                      "letter-spacing": "normal",
                      "margin-top": "30px",
                      "text-align": "left",
                      "width": "33px",
                    }
                  }
                >
                  Email
                   
                </label>
                <input
                  className="form-control"
                  id="IdEmail"
                  name="email"
                  placeholder="Email"
                  type="email"
                />
              </div>
              <div
                className="form-group"
              >
                <label
                  className=""
                  htmlFor="email"
                  style={
                    Object {
                      "color": "#9da5c9",
                      "fontSize": "12px",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                      "height": "12px",
                      "letter-spacing": "normal",
                      "margin-top": "30px",
                      "text-align": "left",
                      "width": "33px",
                    }
                  }
                >
                  Name
                   
                </label>
                <input
                  className="form-control"
                  id="IdEmail"
                  name="name"
                  placeholder="Name"
                  type="email"
                />
              </div>
              <div
                className="form-group"
              >
                <label
                  className=""
                  htmlFor="password"
                  style={
                    Object {
                      "color": "#9da5c9",
                      "fontSize": "12px",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                      "height": "12px",
                      "letter-spacing": "normal",
                      "margin-top": "30px",
                      "text-align": "left",
                      "width": "75px",
                    }
                  }
                >
                  Password
                </label>
                <input
                  className="form-control"
                  id="IdPassword"
                  name="password"
                  placeholder="Password"
                  type="password"
                />
              </div>
            </div>
            <div>
              <button
                className="btn btn-info"
                onClick={[Function]}
                style={
                  Object {
                    "background-image": "linear-gradient(to left, #57aad1, #7a57d1)",
                    "border-radius": "100px",
                    "height": "54px",
                    "width": "192px",
                  }
                }
                type={undefined}
              >
                Continue
              </button>
            </div>
            <div
              style={
                Object {
                  "color": "#3b425d",
                  "fontSize": "16px",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                  "height": "16px",
                  "letter-spacing": "normal",
                  "margin-top": "120px",
                  "text-align": "left",
                  "width": "300px",
                }
              }
            >
              <p>
                Already have an account ?
                <a
                  className="nav-link"
                  href="#"
                  onClick={[Function]}
                  style={
                    Object {
                      "display": "inline",
                    }
                  }
                >
                  Sign in
                </a>
              </p>
            </div>
          </form>
        </div>
      </div>
      <div
        className="col-md-8"
      >
        <div
          style={
            Object {
              "backgroundImage": "url('https://www.onecomputer.fr/wp-content/uploads/2016/01/blurred-background-81.jpg')",
              "height": "100vh",
              "position": "relative",
              "width": "max-width",
            }
          }
        >
          <div
            className="container-fluid"
          >
            <div
              className="row"
            >
              <div
                className="col-9"
              >
                <p
                  style={
                    Object {
                      "color": "#3b425d",
                      "font-size": "16px",
                      "font-style": "normal",
                      "font-weight": "normal",
                      "letter-spacing": "normal",
                      "margin-left": "43vh",
                      "margin-top": "49px",
                      "position": "absolute",
                      "text-align": "left",
                    }
                  }
                >
                  Already have an account ?
                </p>
              </div>
              <div
                className="col-3"
              >
                <button
                  className="btn btn-outline-secondary"
                  onClick={[Function]}
                  style={
                    Object {
                      "border": "solid 1px #7a57d1",
                      "borderRadius": "100px",
                      "color": "#7a57d1",
                      "height": "54px",
                      "margin-top": "30px",
                      "width": "156px",
                    }
                  }
                  type={undefined}
                >
                  Sign in
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </form>
</div>
`;
